code/__DEFINES/dcs/signals.dm

```diff
@@ -335,7 +335,9 @@
 #define COMSIG_MOB_DEADSAY "mob_deadsay"
 	#define MOB_DEADSAY_SIGNAL_INTERCEPT (1<<0)
 ///from /mob/living/emote(): ()
-#define COMSIG_MOB_EMOTE "mob_emote"
+#define COMSIG_MOB_EMOTE "mob_emote"  // todo remove this
```

Comment:
This won't actually be getting removed, it should still be fired for the general emote case.